### PR TITLE
Note about parameters

### PR DESCRIPTION
**Pull request by humbertcostas**
Incremented documentation details.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@humbertcostas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@humbertcostas Can you also add this to the metadata.description of the parameter?

**Comment by azurecla** [comment]
Hi __@humbertcostas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: docker-wordpress-mysql/azuredeploy.parameters.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
       "value": "password"
     },
     "dnsNameForPublicIP": {
-      "value": "uniqueDNS"
+      "value": "uniquedns",
+      "metadata": {
```
Comment (singhkay) [inline docker-wordpress-mysql/azuredeploy.parameters.json]: @humbertcostas This goes in the azuredeploy file :)

**Comment by singhkay** [comment]
@humbertcostas The metadata.description goes in the azuredeploy.json file. Are you making this change to just one template or all templates?

**Comment by humbertcostas** [comment]
Hi @singhkay , I added the specific metadata description for dnsNameForPublicIP in all azuredeploy.json files that was defined. I also revised all azuredeploy.parameters.json files changing values that won't match. Now Travis says that still has errors, but I see that all errors are for other reasons. I think that you could commit my changes and create a new issue to solve the new errors. I'm working with @maneta , who will be happy to work on it.

**Comment by singhkay** [comment]
@humbertcostas Made a task about DNS names and the best guidance around this. See here https://github.com/Azure/azure-quickstart-templates/issues/966

**Comment by azurecla** [comment]
__@humbertcostas__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by humbertcostas** [comment]
Hi @singhkay , I signed the CLA. Do you need somethign else?

**Comment by singhkay** [comment]
@humbertcostas  For the dns parameter names see the guidance in #966 and for parameters file, see the guidance on enabling TravisCI https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci

Can you make these updates?

**Comment by singhkay** [comment]
@humbertcostas closing due to lack of activity. Feel free to reopen with an update

**Pull request closed without merging**